### ISSUE TITLE
Failed to apply plugin [class 'com.soywiz.korge.gradle.KorgeGradlePlugin']

### ISSUE THREAD

**Issue by fatihkizmaz** [korlibs/korge-hello-world#8]
I tried to start hello world project. 
But got an error:  

> Build file 'F:\gameProjects\korge-hello-world-master\build.gradle.kts' line: 23
> 
> Failed to apply plugin [class 'com.soywiz.korge.gradle.KorgeGradlePlugin']
> \> Configuration with name 'jvmMainImplementation' not found.

![image](https://user-images.githubusercontent.com/17065192/80864572-44743d00-8c8c-11ea-83cf-ce754bca7e26.png)

**Comment by soywiz** [comment]
Can you execute:

```
java -version
```

I have

```
java -version
openjdk version "1.8.0_252"
OpenJDK Runtime Environment (AdoptOpenJDK)(build 1.8.0_252-b09)
OpenJDK 64-Bit Server VM (AdoptOpenJDK)(build 25.252-b09, mixed mode)
```

Which intellij version are you using?

Can't reproduce that error

**Comment by fatihkizmaz** [comment]
java

> java version "14.0.1" 2020-04-14
> Java(TM) SE Runtime Environment (build 14.0.1+7)
> Java HotSpot(TM) 64-Bit Server VM (build 14.0.1+7, mixed mode, sharing)

intellij

> IntelliJ IDEA 2020.1.1 (Ultimate Edition)
> Build #IU-201.7223.91, built on April 30, 2020
> Licensed to IntelliJ IDEA Evaluator
> Expiration date: June 1, 2020
> Runtime version: 11.0.6+8-b765.40 amd64
> VM: OpenJDK 64-Bit Server VM by JetBrains s.r.o
> Windows 10 10.0
> GC: ParNew, ConcurrentMarkSweep
> Memory: 2004M
> Cores: 4
> Non-Bundled Plugins: com.soywiz.korge.korge-intellij-plugin

os

> Win10 x64

**Comment by soywiz** [comment]
Was unable to reproduce it with a cleanly installed windows 10 1909 ENG x64. With IntelliJ community and the korge plugin. Installing JDK from the IDE.

Can you open a CMD and execute `systeminfo`? (Remember to remove Registered Owner and Registered Organization from the output).

The output looks like this:
![](https://i2.wp.com/www.pctechbytes.com/wp-content/uploads/2010/12/systeminfo.jpg?fit=600%2C517&ssl=1)

Maybe it is a weird bug related to locale or something?

So:

```
systeminfo
```

The other thing that could help (maybe there is something useful there too):

```
gradlew -i --stacktrace runJvm
```

**Comment by soywiz** [comment]
After some chat on Slack, it seems that the error happens when the operating system language is in Turkish. Since I have it in english I'm going to investigate to switch to Turkish and other languages to see if I can figure out why is it happening.

Thanks for the report!

**Comment by soywiz** [comment]
"Fixed" in `1.12.2.4` here: https://github.com/korlibs/korge-plugins/commit/1feafea0016212d2126862f9278fbef824d88074
It is a hack. Probably either Kotlin or Gradle did something wrong.

Instead of `jvmMainImplementation` with the operating system in Turkish it is creating `jvmMainİmplementation`.
So I bet Gradle `6.2.x` `6.3` or Kotlin `1.3.72` has something wrong there that must be reported because other plugins and code might rely on the `Implementation` part

**Comment by soywiz** [comment]
Reported here: https://youtrack.jetbrains.com/issue/KT-38712

**Comment by YektaDev** [comment]
Although this is pretty old, the issue is fixed in Kotlin `1.8.20-Beta`. It's safe to remove the "hack"s after an upgrade.

**Comment by soywiz** [comment]
> "Fixed" in `1.12.2.4` here: [korlibs/korge-plugins@1feafea](https://github.com/korlibs/korge-plugins/commit/1feafea0016212d2126862f9278fbef824d88074) It is a hack. Probably either Kotlin or Gradle did something wrong.
> 
> Instead of `jvmMainImplementation` with the operating system in Turkish it is creating `jvmMainİmplementation`. So I bet Gradle `6.2.x` `6.3` or Kotlin `1.3.72` has something wrong there that must be reported because other plugins and code might rely on the `Implementation` part

I'm not sure that it will be that easy. Since that's a gradle plugin, and Gradle includes Kotlin unless I'm wrong, the gradle version determines the Kotlin version and thus not sure if the fix will come anytime soon. At some point we will be able to enforce a gradle version to use the korge plugin, but in the meantime that hack might need to be there.

Edit: Still that method is probably not going to be used too much and might be deprecated in favour of kproject
